components/textfield/index.css

```diff
@@ -410,7 +410,8 @@ governing permissions and limitations under the License.
 	}
 
 	/* keyboard focus */
-	.is-keyboardFocused & {
+	.is-keyboardFocused &,
+	&:focus-visible {
```

Comment:
The way textfield was before, it didn't have that nice focus outline when you keyboard focus, and that didn't seem quite right.